mcs/class/System.Net.Http/System.Net.Http/HttpClient.cs

```diff
@@ -41,6 +41,7 @@ public partial class HttpClient : HttpMessageInvoker
 		CancellationTokenSource cts;
 		bool disposed;
 		HttpRequestHeaders headers;
+		HttpMessageHandler handler;
```

Comment:
The base type already have the handler stored

Reply:
It is inaccessible there.  There choice we have is one of the following:

1. Add a private field to the legacy handler that's eventually getting removed anyways.
2. Add Mono-specific hack here: https://github.com/mono/corefx/blob/master/src/System.Net.Http/src/System/Net/Http/HttpMessageInvoker.cs#L16
3. Discuss with the CoreFX team why exactly 2. should be upstreamed

Reply:
I think this code is used only for "legacy" mode where we have all sources in mcs, making the base field `private protected` should then just work